components/engine_test/src/lib.rs

```diff
@@ -343,17 +334,22 @@ pub mod kv {
         }
 
         #[inline]
-        fn tablet_path(&self, id: u64, suffix: u64) -> PathBuf {
+        fn tablet_path_with_prefix(&self, id: u64, suffix: u64, prefix: &str) -> PathBuf {
             self.inner
                 .root_path
-                .join(format!("tablets/{}_{}", id, suffix))
+                .join(format!("tablets/{}{}_{}", prefix, id, suffix))
```

Comment:
Better in format `split_id_suffix`.

Reply:
If {}_{}_{} is used, and if the prefix is "", it will be _x_y.

Reply:
Isn't solved by a simple if/else? The point is `split_id_suffix` is a better format, not to do it with one line.

Though you can also do it by one line like
```
format!("{prefix}{:_>width$}{id}_{suffix}", "", width = !prefix.is_empty() as usize);
```

Reply:
As this method is used in special situations, I predefined the prefix

`pub const SPLIT_PREFIX: &str = "split_";`
`pub const MERGE_PREFIX: &str = "merge_";`

and it is called by using these predefined const str rather than passing a arbitrary str.

Reply:
This works too.